github4s/shared/src/main/scala/github4s/Decoders.scala

```diff
@@ -265,6 +265,25 @@ object Decoders {
       )
     )
 
+  implicit val decodePublicRepositoryEvent: Decoder[PublicRepositoryEvent] =
```

Comment:
`PublicRepositoryEvent` model looks similar to `PublicOrganizationEvent`. Can you please check the API and see if we can reuse the model?

Reply:
Yes they share the same schema, we can merge both models into the same one